### PR TITLE
feat: database migration

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#368886](https://dev.sun-asterisk.com/issues/368886)

## What's this PR do ?

- [x] Migrate database and init model

## Library
*(List package, library third party add new include version)*

- laravel-enum

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [ ] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [ ] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*

### CODE REVIEW

File: database/migrations/2019_11_19_060449_create_webhooks_table.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('webhooks', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('user_id')->unsigned();
+            $table->bigInteger('bot_id')->unsigned();
+            $table->string('token');
```
Comment (ghost) [inline database/migrations/2019_11_19_060449_create_webhooks_table.php]: unique

File: database/migrations/2019_11_19_060449_create_webhooks_table.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class CreateWebhooksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('webhooks', function (Blueprint $table) {
```
Comment (ghost) [inline database/migrations/2019_11_19_060449_create_webhooks_table.php:17]: thiếu cái webhook name rồi thì phải

File: database/migrations/2019_11_19_060606_create_payload_histories_table.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('payload_histories', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('webhook_id')->unsigned();
+            $table->text('params')->nullable();
+            $table->integer('status');
```
Comment (phucqd-1305) [inline database/migrations/2019_11_19_060606_create_payload_histories_table.php:20]: Status này chưa có Enum

File: database/migrations/2019_11_19_060619_create_message_histories_table.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('message_histories', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('payload_history_id')->unsigned();
+            $table->text('message_content');
+            $table->integer('status');
```
Comment (phucqd-1305) [inline database/migrations/2019_11_19_060619_create_message_histories_table.php]: Status này nữa

File: database/migrations/2019_11_19_060606_create_payload_histories_table.php (hunk 3 above)
Comment (ghost) [reply database/migrations/2019_11_19_060606_create_payload_histories_table.php:20]: ý a có phải là chưa set default value cho nó ko a?
còn nếu enum cho payload_history_status e có tạo đây rồi á a
https://github.com/sun-asterisk-research/chatwork-forwarder/pull/3/files#diff-3a9ba6bef0898664d21972db8a033a9eR12

Comment (phucqd) [reply database/migrations/2019_11_19_060606_create_payload_histories_table.php:20]: ok em, cái này thì k cần default, vậy đúng rồi :D

**Review by quanghung97 — APPROVED** [review]
OK

**Review by nguyenthinhhn — APPROVED** [review]

**Review by tamtt-1031 — APPROVED** [review]

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**